package.json

```diff
@@ -32,9 +32,9 @@
     "streamqueue": "^1.1.2",
     "systemjs": "^0.21.5",
     "systemjs-plugin-babel": "^0.0.25",
+    "terser": "^3.10.12",
     "ttest": "^1.1.0",
     "typogr": "^0.6.8",
-    "uglify-es": "^3.3.9",
```

Comment:
It looks like it's not removed from the `package.json` properly; refer to https://github.com/mozilla/pdf.js/pull/10292/files#diff-32607347f8126e6534ebc7ebaec4853dR13111.

Reply:
It's not removed because outdated Webpack still uses it. Once you upgrade Webpack to `>=4.26.0`, it will disappear from `package-lock.json`.